packages/client/src/components/PrintDetails.tsx

```diff
@@ -17,6 +17,7 @@ const currentDateTime = date.toLocaleDateString("nl-NL", options);
 export default () => (
   <PrintOnly>
     <Heading forwardedAs="h3">Pagina</Heading>
+    {/* TODO: hide ?data=... from url */}
```

Comment:
You can use `removeQueryStrings(window.location.href)` to remove everything after the `?` from a URI (import from `packages/client/src/utils/index.ts`)